components/header.stories.js

```diff
@@ -0,0 +1,8 @@
+import './header.css'
+import kopf from './header.html'
+
+export default {
+  title: 'Components',
```

Comment:
Here, I would suggest to add title and after that just html for the component. Something like that:

```suggestion
  title: 'Components',
  
  
  import './header.css'
import html from './header.html'

export default {
  title: 'Header',
  component: html,
}

export const base = () => html

  
```